File: A5/src/command/RunExample.java
package command;
import java.io.IOException;

import controller.*;
import model.MyStmtException;
public class RunExample extends Command
{
	private Controller ctrl;
	public RunExample(String key, String desc, Controller c)
	{
		super(key, desc);
		this.ctrl = c;
	}
	@Override
	public void execute() throws IOException, MyStmtException
	{
		ctrl.allStep();
	}
}

File: A7/Model/RunExample.cs
using A7.Ctrl;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace A7.Model
{
    class RunExample: Command
    {
        private Controller ctrl;
        String key, desc;
        public RunExample(String key, String desc, Controller c): base(key, desc)
        {
            this.ctrl = c;
        }
        public override void Execute()
        {
            ctrl.AllStep();
        }
}
}
